README.md

```diff
@@ -169,7 +169,9 @@ When you run `azd up` after and are prompted to select a value for `openAiResour
   then run `azd env set AZURE_SEARCH_SERVICE_LOCATION {Location of existing service}`
 1. If the search service's SKU is not standard, then run `azd env set AZURE_SEARCH_SERVICE_SKU {Name of SKU}`. The free tier won't work as it doesn't support managed identity. ([See other possible values](https://learn.microsoft.com/azure/templates/microsoft.search/searchservices?pivots=deployment-language-bicep#sku))
 1. If you have an existing index that is set up with all the expected fields, then run `azd env set AZURE_SEARCH_INDEX {Name of existing index}`. Otherwise, the `azd up` command will create a new index.
-
+1. If you want to configure the language of the search query, then run `azd env set AZURE_SEARCH_QUERY_LANGUAGE {Name of query language}`. ([See other possible values](https://learn.microsoft.com/en-us/python/api/azure-search-documents/azure.search.documents.models.querylanguage?view=azure-python-preview))
+1. If you want to configure the type of the speller to use to spell-correct, then run `azd env set AZURE_SEARCH_QUERY_SPELLER {Name of speller type}`. ([See other possible values](https://learn.microsoft.com/en-us/python/api/azure-search-documents/azure.search.documents.models.queryspellertype?view=azure-python-preview))
+1. If you want to configure the name of the analyzer to use for a searchable text field, then run `azd env set AZURE_SEARCH_ANALYZER_NAME {Name of analyzer name}`. ([See other possible values](https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.search.models.field.analyzer?view=azure-dotnet-legacy&viewFallbackFrom=azure-dotnet))
```

Comment:
These instructions are under "Use existing Azure search resource" which isn't the exact right heading, but perhaps is close enough.

I have some suggested rewording though, to make it clear what the change/defaults are:

```
You can also customize the search service (new or existing) for non-English searches:

1. To configure the language of the search query to a value other than "en-us", run `azd env set AZURE_SEARCH_QUERY_LANGUAGE {Name of query language}`. ([See other possible values](https://learn.microsoft.com/python/api/azure-search-documents/azure.search.documents.models.querylanguage?view=azure-python-preview))
1. To turn off the spell checker, run `azd env set AZURE_SEARCH_QUERY_SPELLER none`. ([See other possible values](https://learn.microsoft.com/python/api/azure-search-documents/azure.search.documents.models.queryspellertype?view=azure-python-preview))
1. To configure the name of the analyzer to use for a searchable text field to a value other than "en.microsoft", run `azd env set AZURE_SEARCH_ANALYZER_NAME {Name of analyzer name}`. ([See other possible values](https://learn.microsoft.com/dotnet/api/microsoft.azure.search.models.field.analyzer?view=azure-dotnet-legacy&viewFallbackFrom=azure-dotnet))
```

Reply:
@pamelafox Thank you for the suggestion. I reflected them [90c6dc4](https://github.com/Azure-Samples/azure-search-openai-demo/pull/780/commits/90c6dc44b4414679ec74a51101dfa5eef4b75362)